### PR TITLE
fix(orchestrator): add lastRunId to overview endpoints

### PR DESCRIPTION
**Pull request by gciavarrini**
* Return last workflow run id in [/workflows/overview](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-backend/src/service/router.ts#L175) and [/workflows/:workflowId/overview](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/orchestrator-backend/src/service/router.ts#L339)
* Update v2 endpoints too.

Address: ([FLPATH-1209](https://issues.redhat.com//browse/FLPATH-1209))[https://issues.redhat.com/browse/FLPATH-1209]

### CODE REVIEW

**Comment by gciavarrini** [comment]
@caponetto @batzionb is it ok to update the backend version?

File: plugins/orchestrator-backend/dist-dynamic/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-orchestrator-backend-dynamic",
-  "version": "1.6.2",
+  "version": "1.6.3",
```
Comment (caponetto) [inline plugins/orchestrator-backend/dist-dynamic/package.json:3]: Please revert this change. Versions are controlled by the semantic release process.

File: plugins/orchestrator-backend/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@janus-idp/backstage-plugin-orchestrator-backend",
-  "version": "1.6.2",
+  "version": "1.6.3",
```
Comment (caponetto) [inline plugins/orchestrator-backend/package.json:3]: Please revert this change. Versions are controlled by the semantic release process.
